modin/core/execution/dask/implementations/pandas_on_dask/io/io.py

```diff
@@ -170,6 +170,19 @@ def to_sql(cls, qc, **kwargs):
         # since the mapping operation is non-blocking, each partition will return an empty DF
         # so at the end, the blocking operation will be this empty DF to_pandas
 
+        import sqlalchemy as sa
```

Comment:
what if `sqlalchemy` is not installed?..

Reply:
Good question, rewritten.